lib/airbrake/rails/app.rb

```diff
@@ -45,7 +45,7 @@ def self.recognize_route(request)
             # versions support.
             #
             # https://github.com/rails/rails/commit/5460591f0226a9d248b7b4f89186bd5553e7768f
-            next if route.respond_to?(:glob?) && route.glob?
+            next if route.nil? && route.respond_to?(:glob?) && route.glob?
```

Comment:
So does the following change with removing `@todo` section above make sense?
```suggestion
            next if route.path.ast.nil? && route.glob?
```